### ISSUE TITLE
[BUG] Windows version is not working

### ISSUE THREAD

**Issue by Ammoton** [stumpapp/stump#440]
**Describe the bug**
If I open stump_server.exe, it opens for a brief second and then closes.

**To Reproduce**

1. Extract the ZIP file.
2. Open stump_server.exe.

**Expected behavior**
The program should open and stay running.

**Screenshots**

**Desktop (please complete the following information):**
- OS: [Windows]

**Additional context**
If I open the program in the terminal, it shows the following error:

```
thread 'main' panicked at D:\a\stump\stump\core\src\db\client.rs:36:10:
Failed to create Prisma client: Connection(ConnectorError { user_facing_error: None, kind: QueryError(SqliteFailure(Error { code: CannotOpen, extended_code: 14 }, Some("unable to open database file: D:\\a\\stump\\stump\\core/prisma/dev.db"))) })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by aaronleopold** [comment]
> unable to open database file: D:\\a\\stump\\stump\\core/prisma/dev.db

Does this file actually exist? This look suspiciously like a non-release path, one I'd expect if you were running from source. 

Have you overridden any of the [configuration options](https://www.stumpapp.dev/guides/configuration/server-options)? If not, this might just be a lingering bug with the config generator. Can you share what your Stump.toml looks like?

**Comment by Ammoton** [comment]
My `Stump.toml` file looked like this:
```
profile = "debug"
port = 10801
verbosity = 0
pretty_logs = true
client_dir = "./dist"
config_dir = 'C:\Users\user\.stump'
allowed_origins = []
disable_swagger = false
password_hash_cost = 12
session_ttl = 259200
expired_session_cleanup_interval = 86400
scanner_chunk_size = 100
```
I changed the profile to release, but it did not work. After that, I deleted the  .stump folder and Stump  and reinstalled Stump, and now it works perfectly, with the profile set to release. I don't know why it didn't work before—perhaps it was because I installed it earlier using Docker Desktop? But thank you very much for your help and for this great project. I will close this issue, as it was most likely my fault.
